docs/reference/nuspec.md

```diff
@@ -303,6 +303,9 @@ Explicit references are typically used for design-time only assemblies. When usi
 
 Similarly, explicit references can be used for unit test frameworks, such as XUnit, which needs its tools assemblies located next to the runtime assemblies, but does not need them included as project references.
 
+> [!Important]
```

Comment:
I am not sure this explanation is detailed enough to be actionable. 
If I were a customer it's unclear to me what the recommendation exactly means here. 
Can you maybe provide an example (in the Samples repo probably), that shows a customer to how to author the package, in a way that makes it works for 
* packages.config
* PackagerReference

Reply:
better?

Reply:
Yeah.  

I still think we should have a samples in the NuGet samples repo :)